hack/ensure-image.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+function ensure::ensure_image_karama_agent() {
```

Comment:
How about `docker::ensure_image_karama_agent` ?

Reply:
docker is better